.github/workflows/deploy_microsite.yml

```diff
@@ -182,7 +182,7 @@ jobs:
     env:
       CI: true
       NODE_OPTIONS: --max-old-space-size=16384
-      DOCUSAURUS_SSR_CONCURRENCY: 5
+      DOCUSAURUS_PERF_LOGGER: true
```

Comment:
should this be on the `verify_microsite.yml` build?

This workflow won't run on PR builds.

Reply:
I don't know, I don't understand anything about your CI setup